internal/dbmodel/usermapping_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2025 Canonical.
+
+package dbmodel_test
+
+import (
+	"database/sql"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+)
+
+// TestUserMapping_UniqueConstraint tests that the unique constraint on
+// UserMapping is enforced correctly ensuring two rows with the same
+// ModelUUID and LocalUser cannot be created.
+func TestUserMapping_UniqueConstraint(t *testing.T) {
```

Comment:
Interesting we using _ here, looks like AI added this because we don't usually?

Reply:
Haha I was wondering if anyone would notice this. I actually added it because I saw AI adding somewhere else in Juju and I liked it. Splits the thing I'm testing with the specifics.

Reply:
I really like it (Imo) but we discussed it once witn Juju and they hated it, and I think I tried to push for it for us and rightfully so, @alesstimec wasn't keen because it's just not how we do things and stood out like a sore thumb. having said that, I'm 1000% for continuing like this if the guys are on board?

Reply:
I don't mind, but since it's different I've just opted to make it consistent now.

Reply:
Aww, we can discuss it another time though I am really in favour of it.